src/posts/gold-design-system.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Celebrating the launch of GOLD Design System 🚀
+date: '2021-09-21'
+tags: [announcements, releases]
+description: GOLD Design System has been published and available to be used in place of the Australian Government Design System   
+permalink: posts/{{ title | slug }}/index.html
+---
+
+Today marks the launch of GOLD Design System, replacing the Australian Government Design System.
```

Comment:
Should that be "the GOLD design system" as in "the government open language for design"?

Reply:
Included the suggestion, plus the next paragraph.